src/sec_limit_analytically.mbx

```diff
@@ -199,7 +199,7 @@
           </li>
 
           <li>
-            <p><m>\lim\limits_{x\to c}a^x = a^c</m> (\text{if } <m>a>0</m>)</p>
```

Comment:
This should end the p with the limit, and have a new p for the condition. If it's all in one m it will be too wide for small screens.

Once you do that, the HTML may look terrible because the CSS for a multi-column list is not yet great. It will be something to hold your nose at for a while.